Build syntect with Rust's regex crate instead of oniguruma (a C library).
The reason is that broot already depends on the regex crate.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -58 +58 @@
-syntect = { package = "syntect-no-panic", version = "4.6.1" } # see issue #485
+syntect = { package = "syntect-no-panic", version = "4.6.1", default-features = false, features = ["default-fancy"] } # see issue #485
